### PR TITLE
createkritipatwa.json

### PR DESCRIPTION
**Pull request by Kritipatwa16**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6494"></a>

### CODE REVIEW

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Your file name must match your GitHub username.  

So instead of data/kritipatwa.json it should be data/Kritipatwa16.json

I would suggest you look at our QuickStart Guide in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request so that we can get it merged.

Your JSON is also failing, check this on: https://jsonlint.com so that you can see how to correct it.

File: data/kritipatwa.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "kriti patwa",
+  "type": "personal",
+  "bio": "college student pursuing btech in cse " 
+  "links": [kriti15038
```
Comment (Sindhuinti) [inline data/kritipatwa.json:5]: remove this `kriti15038`

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

**Review by SaraJaoude — DISMISSED** [review]
Please make sure your file name matches your GitHub username exactly (it is case sensitive). Instead of `data/kritipatwa.json`   it should be `data/Kritipatwa16.json`

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

File: data/kritipatwa.json (hunk 1 above)
Comment (SaraJaoude) [reply data/kritipatwa.json:5]:
```suggestion
  "links": [
```

**Pull request merged**